charts/gardener-extension-shoot-oidc-service/templates/service.yaml

```
@@ -10,13 +10,15 @@ metadata:
   namespace: {{ .Release.Namespace }}
   annotations:
     networking.resources.gardener.cloud/from-world-to-ports: '[{"protocol":"TCP","port":{{ .Values.webhookConfig.serverPort }}}]'
-    networking.resources.gardener.cloud/from-policy-pod-label-selector: all-seed-scrape-targets
-    networking.resources.gardener.cloud/from-policy-allowed-ports: '[{"port":{{ .Values.metrics.port }},"protocol":"TCP"}]'
+    networking.resources.gardener.cloud/from-all-seed-scrape-targets-to-ports: '[{"protocol":"TCP","port":{{ .Values.webhookConfig.serverPort }}}]'
```

```suggestion
    networking.resources.gardener.cloud/from-all-seed-scrape-targets-to-ports: '[{"protocol":"TCP","port":{{ .Values.metrics.port }}}]'
```

I guess this should also be changed